app/styles/main.css

```diff
@@ -889,13 +889,12 @@ footer a:hover, footer a:focus, footer a:active {
 .btn {
```

Comment:
on touch interfaces, touching the button results in a weird state with black text and a weird outline:
![img_3260](https://user-images.githubusercontent.com/597182/36819979-f4df8928-1ceb-11e8-8c78-35ba8be54a98.png)


on an iphone-size screen, the text and content being left justified looks weird - especially the app buttons - can we try centering those?